includes/Modules/Analytics.php

```diff
@@ -589,6 +585,43 @@ protected function get_datapoint_services() {
 		);
 	}
 
+	/**
+	 * Gets map of datapoint to definition data for each.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array Map of datapoints to their definitions.
+	 */
+	protected function get_datapoint_definitions() {
+		$map = parent::get_datapoint_definitions();
+
+		$map['POST:create-account-ticket'] = array_merge(
+			$map['POST:create-account-ticket'],
+			array(
+				'scopes'                 => array( 'https://www.googleapis.com/auth/analytics.provision' ),
+				'request_scopes_message' => __( 'Additional permissions are required to create a new Analytics account.', 'google-site-kit' ),
+			)
+		);
+
+		$map['POST:create-property'] = array_merge(
+			$map['POST:create-property'],
+			array(
+				'scopes'                 => array( 'https://www.googleapis.com/auth/analytics.edit' ),
+				'request_scopes_message' => __( 'Additional permissions are required to create a new Analytics property.', 'google-site-kit' ),
+			)
+		);
+
+		$map['POST:create-profile'] = array_merge(
+			$map['POST:create-profile'],
+			array(
+				'scopes'                 => array( 'https://www.googleapis.com/auth/analytics.edit' ),
+				'request_scopes_message' => __( 'Additional permissions are required to create a new Analytics view.', 'google-site-kit' ),
+			)
+		);
```

Comment:
Note that `POST:settings` _may_ currently require `https://www.googleapis.com/auth/analytics.edit` too. I don't think we're using the datapoint this way anymore though (we no longer pass non-valid property ID/profile ID there to create them right?), so I think it'd be better to just remove that functionality rather than adjust for the legacy edge-case here.

Reply:
I didn't realize that was still there. The refactored components only use the dedicated datapoints so I'll remove the old behavior from `POST:settings`. I think it doesn't even need the settings datapoint in there at all since this is handled by `Modules` now.

One thing we were doing in the settings datapoint that we aren't in the new datapoints is when we created a new property and profile, we set the profile as the default profile for that property. 

I've removed the old settings datapoint handler for now. I think we can look at restoring this functionality in a follow-up issue, if necessary. I wonder if Analytics makes it the default view anyways since it's the first one created?

Reply:
That's a good catch, let's test that. If it doesn't, we should fix this because it would be a regression. It would be weird if you created a new property and profile and then that new profile wouldn't be the one selected.

Reply:
It would still be selected because if a property does not have a `defaultProfileId` then we select the first, so it shouldn't be a regression in terms of UX.

If we wanted to add support for this to the `create-profile` datapoint, we would have to add a new parameter or something as we would only want to do this when the two are created together, not every time we create a new profile.

Is that something you feel we need to add right now? Technically, this is already the case with 1.8.

I'm not sure it's really necessary though, as it would only make a difference if the user created both a property and a profile, and then created another profile, but expected the first to be selected by default (which would probably happen anyways due to the current logic). It seems like an extreme edge case. Is there more to this that I'm missing?